projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -90,11 +90,19 @@ public function enqueue_admin_scripts() {
 				'textdomain' => 'jetpack-social',
 			)
 		);
+
+		global $publicize;
+		wp_localize_script(
+			'jetpack-social',
+			'jetpackSocial',
+			array(
+				'connections' => $publicize->get_all_connections_for_user(),
+			)
+		);
 		Assets::enqueue_script( 'jetpack-social' );
 		// Initial JS state including JP Connection data.
 		wp_add_inline_script( 'jetpack-social', Connection_Initial_State::render(), 'before' );
 		wp_add_inline_script( 'jetpack-social', $this->render_initial_state(), 'before' );
```

Comment:
This is where we should be adding the `connections` info instead (the `$this->initial_state` function).